pkg/server/version.go

```diff
@@ -18,13 +18,21 @@ package server
 
 import (
 	"errors"
+	"fmt"
 
 	"golang.org/x/net/context"
 
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+var containerdVersion = "UNKNOWN"
```

Comment:
ditto. This is not containerdVersion.

And I prefer putting the logic into `pkg/version/version.go`. This file is to implement the `Version` function defined in CRI, which returns the runtime version.